source/class/qx/test/event/GlobalError.js

```diff
@@ -198,7 +198,7 @@ qx.Class.define("qx.test.event.GlobalError", {
       window.onerror = function (msg, uri, lineNumber) {
         wasNativeHandled = true;
 
-        self.assertMatch(msg, "Doofer Fehler");
+        self.assertMatch(msg, "Doofer Fehler 2");
```

Comment:
I don't understand this change

Reply:
It was the same message as used in a different test, and I needed to distinguish between the two when one of them was failing.